engine/src/test/java/org/hibernate/validator/test/internal/constraintvalidators/hv/SafeHtmlValidatorTest.java

```diff
@@ -170,4 +182,16 @@ public Foo(String source) {
 			this.source = source;
 		}
 	}
+
+	public static class Bar {
+		@SafeHtml(
+				whitelistType = WhiteListType.BASIC,
+				additionalTagsWithAttributes = @SafeHtml.Tag(name = "img", attributes = "src", protocols = { "data" })
```

Comment:
Would be good to have more than one element in the test, so we know that works, too :)